_includes/v22.2/connect/connection-url.md

```diff
@@ -0,0 +1,19 @@
+<div class="filter-content" markdown="1" data-scope="mac linux">
+Set a `DATABASE_URL` environment variable to your connection string.
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+export DATABASE_URL="{connection string}"
+~~~
```

Comment:
Setting the env variable at the top of the connect section is a nice improvement!

Did you implement it by using a placeholder and an include to simplify what edits would be required for this revision (as opposed to using the connection string template as the value of DATABASE_URL in each data-scope)?

Reply:
Yes. The actual connection string will differ based on the deployment, so I added info about how to find the connection string earlier, when the user selects their deployment.

The vast majority of the examples can use a connection string (exceptions are Java, which uses a JDBC connection string, and Django, which requires parameters). So we have includes for general connection strings and JDBC connection strings, and don't need anything else for Django.

Reply:
Thanks for wrapping this up @ianjevans! LGTM

Reply:
@ianjevans apologies for the delayed follow up. I didn't see your response because I had notifications going to the wrong email address. Fixed now, but sorry to keep you waiting. I see this was LGTMed and merged. Great!